.circleci/config.yml

```diff
@@ -350,9 +355,9 @@ jobs:
       - run:
           name: build js types
           command: |
-            mkdir protos
-            ./node_modules/@textile/protobufjs/bin/pbjs -t static-module -w es6 -o protos/index.js ./pb/protos/*
-            ./node_modules/@textile/protobufjs/bin/pbts -o protos/index.d.ts protos/index.js
+            mkdir js-types
+            ./node_modules/@textile/protobufjs/bin/pbjs -t static-module -w es6 -o js-types/index.js ./pb/protos/*
+            ./node_modules/@textile/protobufjs/bin/pbts -o js-types/index.d.ts js-types/index.js
```

Comment:
Changed the directory name here so it doesn't conflict with other files in the workspace persistence